config/sources/families/bcm2711.conf

```diff
@@ -48,14 +48,9 @@ case "${BRANCH}" in
 esac
 
 function post_family_config__mark_bsp_as_firmware_containing() {
-	if [[ "${SKIP_ARMBIAN_REPO}" != "yes" ]]; then
-		declare -g -r RASPI_FIRMWARE_DIR="/usr/lib/linux-firmware-raspi"
-		display_alert "SKIP_ARMBIAN_REPO is set to" "${SKIP_ARMBIAN_REPO} - not adding rpi firmware to bsp" "info"
-		return 0
-	fi
 	display_alert "Marking BSP name as firmware-containing" "bcm2711" "info"
 	declare -g EXTRA_BSP_NAME="${EXTRA_BSP_NAME}-raspifw" # Unique bsp name.
-	declare -g -r RASPI_FIRMWARE_DIR="/usr/lib/armbian-firmware-raspi"
+	declare -g -r RASPI_FIRMWARE_DIR="/usr/lib/linux-firmware-raspi"
```

Comment:
Either revert this variable value change, or also make changes to `declare destdir="${destination}/usr/lib/armbian-firmware-raspi"` in `post_family_tweaks_bsp__rpi_firmware_in_bsp` function

Reply:
Fixed, but Jammy works, Bookworm "missing firmware".

Reply:
Hmm. Will take a look tomorrow

Reply:
I'll keep looking for a while ...

Reply:
Both works this way, but I am not fully happy yet.